test/smoke/testApps/InheritedAttributes/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootAutoTest.java

```diff
@@ -60,6 +60,6 @@ public void test() throws Exception {
     assertEquals("smoketestapp", md.getProperties().get("LoggerName"));
     assertNotNull(md.getProperties().get("ThreadName"));
     assertEquals("z", rd.getProperties().get("tenant"));
-    assertEquals(5, md.getProperties().size());
+    assertEquals(4, md.getProperties().size());
```

Comment:
I don't remember why this change?

Reply:
i checked main branch; 

it returned. 

`{"LoggerName":"smoketestapp","LoggingLevel":"INFO","SourceType":"Logger","ThreadName":"http-nio-8080-exec-2","tenant":"z"}`

```
ThreadContext.getRequestTelemetryContext()
        .getHttpRequestTelemetry()
        .getProperties()
        .put("tenant", "z");
```

Reply:
why doesn't the same thing happen in this PR branch? just want to make sure we didn't inadvertently break something